services/user/Tokens/plugin/wit/world.wit

```diff
@@ -0,0 +1,51 @@
+package component:tokens;
+
+
+
+interface types {
+
+    type quantity = string;
+    type precision = u8;
+    type tid = u32;
+    type account-number = string;
+
+    record balance {
+        balance: string,
+        creditor: string,
+        debitor: string,
+        token-id: u32,
```

Comment:
Users as in front end devs? We still want them to be mindful of `symboless`  tokens don't we? Especially in cases where apps are created and tokens are in use but completely abstracted away from users.

From the front-end dev side, it's easier to work with a `ID` property you know you're always gonna have rather than the `symbol` one you might have. 

I see a lookup case where users type in a symbol to select a token, but a query will still be necessary to learn it's precision along with any other relevant attributes to the app. Fairly rare to work with a token without it's precision.

I feel if we force users to interface with this by ID only it will also create a side effect of leading dapp devs to center around `ID` instead of `symbol` use, so hopefully they're less likely to build a `symbol` orientated app first then `symboless` tokens are some awkward after thought.

I know plugins should lift the weight of front end devs, but I feel this particular burden is worth putting on the frontend devs shoulders.

Reply:
Vast majority of front-ends that interface with tokens probably should not support symbolless tokens, it will only needlessly complicate the UI.  Only rare edge cases (admin pages or apps still under construction) should interact with symbolless tokens.

My new proposal: 
```
type token-id: string;
```
This interface would support either symbols or token ids being passed to it. Since SIDs cannot start with a numeric digit, and TIDs cannot start with a letter, the plugin can simply parse the first character of the string to determine what is being used. Most front-ends can simply work with symbols, but if a front-end wants to, they can operate using token ids directly.

Reply:
Makes sense, I'll add this. 

>  "Only rare edge cases (admin pages or apps still under construction) should interact with symbolless tokens."

I see a potential use case in fractally, if a fractal doesn't yet have a symbol or I don't care whatever the symbol is I'd just want to search the fractal by its name and buy it's token, can be completely ignorant to whatever the symbol is. 

Dan also proposed a potential social app that had users have their own token out of the gate for others to invest in, but on the front end users remain oblivious that there's actually a token behind the scenes.

Reply:
Also DeFi apps like relay tokens, probs don't need a symbol either.